src/Command/Compile/Options.hs

```diff
@@ -9,4 +9,5 @@ data Options = Options
   , optimisation :: Maybe String
   , assemblyDir :: Maybe FilePath
   , llvmConfig :: Maybe FilePath
+  , extraLibDir :: Maybe FilePath
```

Comment:
Can this be a `[FilePath]` instead so we can specify multiple paths? I think you can use `many` instead of `optional` in the command-line parser to make that work.

Reply:
Fixed. Thanks for your suggestions!